tests/unit/build/CMakeLists.txt

```
@@ -49,15 +49,18 @@ function(
     )
   endif()
   set(test_dir "${PROJECT_SOURCE_DIR}/${test_dir}")
+  if(HPX_WITH_CLANG_CUDA)
+    set(cmake_cuda_compiler -DCMAKE_CUDA_COMPILER=$ENV{CXX})
```

```suggestion
    set(cmake_cuda_compiler -DCMAKE_CUDA_COMPILER=${CMAKE_CUDA_COMPILER})
```
?

`CXX` is not necessarily set.